src/setup_test.ts

```diff
@@ -76,8 +77,10 @@ const EXCLUDE_LIST: string[] = [
   'concat tensors with 0 in their shape',                // Timeout.
   'argmax test-webgpu {} accepts tensor with bool',      // Actual != Expected.
   'encodeWeights',                                       // Bool tensors fails.
-  'deprecation warnings',  // tf.disableDeprecationWarnings.
-  'oneHot',                // Not yet implemented
+  'deprecation warnings',             // tf.disableDeprecationWarnings.
+  'oneHot',                           // Not yet implemented.
+  'accepts tensor with bool values',  // Error happens in ANGLE.
```

Comment:
Please file an angle bug to track it. And past the link here.

Reply:
This root cause is still in tfjs-compute, currently, bool type in tensor is not yet implemented.